doc/radosgw/dynamicresharding.rst

```diff
@@ -37,8 +37,8 @@ resharding tasks, one at a time.
 Multisite
 =========
 
-Dynamic resharding is not supported in a multisite environment.
-
+Prior to the Reef release, RADOSGW does not support dynamic resharding in a
```

Comment:
Sometimes we call it RGW, sometimes recently Ceph Object Gateway, but it shouldn't be all uppercase.

Reply:
Good looking out.